content/docs/reference-react-dom-server.md

```diff
@@ -49,9 +49,9 @@ If you call [`ReactDOM.hydrate()`](/docs/react-dom.html#hydrate) on a node that
 ReactDOMServer.renderToStaticMarkup(element)
 ```
 
-Similar to [`renderToString`](#rendertostring), except this doesn't create extra DOM attributes that React uses internally, such as `data-reactroot`. This is useful if you want to use React as a simple static page generator, as stripping away the extra attributes can save some bytes.
+דומה ל-[`renderToString`](#rendertostring), מלבד העובדה שאינה יוצרת מאפייני DOM נוספים ש-React משתמשת בהם באופן פנימי, כגון `data-reactroot`. אפשרות זו שימושית אם ברצונך להשתמש ב-React כמחולל של דפים סטטיים פשוטים, שכן הסרת המאפיינים הנוספים יכולה לחסוך כמה בתים.
```

Comment:
According to Morfix, there are two options for [byte](https://www.morfix.co.il/byte), בית or בייט. I know בית is the Hebrew word but בייט sounds more clear to me. Tell me what do you think about it.

Reply:
As far as I know, in most literature I read בית is usually used, this is why I used it as well, also I think it sounds better in plurals. I think it is better than the other option, but if there's a consensus I wouldn't mind changing.

Reply:
At first, it confused me, but after I read it twice it was understandable. It doesn't mind to me, but I know it'll be clearer to others.